test/scripts/openshift-ci/deploy.serverless.sh

```diff
@@ -144,12 +147,9 @@ spec:
     - hosts:
         - '*'
       port:
-        name: https
-        number: 443
```

Comment:
isn't better to keep ssl to test self signed certificates?

Reply:
I'd like to say yes, but the upstream E2E tests don't work with TLS.

We probably should contribute a basic test suite that has ensures self-signed TLS works.

Reply:
Thinking twice, testing TLS here doesn't make too much sense. The point where makes sense in the storage initializer fetching from self signed storage.